src/component/feature/FeatureToggleList/FeatureToggleArchiveList.tsx

```diff
@@ -101,26 +98,9 @@ export const FeatureToggleList: VFC<IFeatureToggleListProps> = ({
                     />
                 ))}
                 elseShow={
-                    <ConditionallyRender
-                        condition={Boolean(isArchive)}
-                        show={
-                            <ListItem className={styles.emptyStateListItem}>
-                                No archived features.
-                            </ListItem>
-                        }
-                        elseShow={
-                            <ConditionallyRender
-                                condition={Boolean(createFeature?.access)}
-                                show={() => (
-                                    <ListPlaceholder
-                                        text="No features available. Get started by adding a new feature toggle."
-                                        link={createFeature?.path}
-                                        linkText="Add your first toggle"
-                                    />
-                                )}
-                            />
-                        }
-                    />
+                    <ListItem className={styles.emptyStateListItem}>
+                        No archived features.
```

Comment:
Should rename the component to match the renamed file.

Reply:
I'll clean this up when I refactor `ArchiveList`. I'm avoiding changing things that I will delete later on.